threshsign/src/bls/relic/BlsMultisigAccumulator.cpp

```diff
@@ -58,7 +58,6 @@ void BlsMultisigAccumulator::sigToBytes(unsigned char * outThreshSig, int thresh
 }
 
 void BlsMultisigAccumulator::aggregateShares() {
-    assertEqual(validSharesBits.count(), reqSigners);
```

Comment:
Why'd you remove this assert?

Reply:
The logic checking that the number of signers is not less than required has moved to BlsMultisigVerifier::verify().
And it's not an assertion anymore, but an error, since previously multisig was used in case of n-out-of-n